File: 151001/Kononchuk/REST/src/REST/Models/Entities/Tag.cs
namespace REST.Models.Entities;

public class Tag
{
    public long Id { get; set; }
    public string Name { get; set; }
    
    public List<Issue> Issues { get; set; } = new();
    
    public List<IssueTag> IssueTags { get; set; } = new();
}

File: 151004/Prokopchuk/labs1-5/src/main/java/by/bsuir/test_rw/model/entity/implementations/Tag.java
package by.bsuir.test_rw.model.entity.implementations;

import by.bsuir.test_rw.model.entity.interfaces.EntityModel;
import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;
import java.util.List;

@Entity
@Setter
@Getter
@Table(name = "tbl_tag")
public class Tag implements EntityModel<Long>, Serializable {
    @GeneratedValue
    @Id
    private Long id;
    private String name;
    @ManyToMany
    private List<Issue> issues;

    @Override
    public Long getId() {
        return id;
    }

    @Override
    public void setId(Long id){
        this.id = id;
    }
}
